spec/index.bs

```diff
@@ -1326,6 +1331,77 @@ dictionary GPUBufferBinding {
+        1. Ensure [=sampler binding validation=] is not violated.
+    1. If |bindingDescriptor|.{{GPUBindGroupBinding/resource}} is {{GPUTextureView}}:
+        1. Ensure it is a valid {{GPUTextureView}} object.
+        1. Ensure [=texture view binding validation=] is not violated.
+    1. If |bindingDescriptor|.{{GPUBindGroupBinding/resource}} is {{GPUBufferBinding}}
+        1. Ensure it is a valid {{GPUBufferBinding}} object.
+        1. Ensure [=buffer binding validation=] is not violated.
+    1. Insert |bindingDescriptor| into |bindGroup|.{{GPUBindGroup/[[bindings]]}}.
+1. Return |bindGroup|.
+
+<b>Validation Conditions</b>
+
+<dl dfn-for="createBindGroup(descriptor)">
+If any of the following conditions are violated:
+    1. Generate a {{GPUValidationError}} in the current scope with appropriate error message.
+    1. Create a new invalid {{GPUBindGroup}} and return the result.
+
+<dfn>bind group device validation</dfn>: The {{GPUDevice}} must not be lost.
+
+<dfn>bind group binding validation</dfn>: Each |bindingDescriptor|.{{GPUBindGroupBinding/binding}} in |descriptor| must be unique.
+
+<dfn>sampler binding validation</dfn>: A corresponding sampler binding should exist in {{GPUBindGroupLayoutDescriptor/bindings}} of |descriptor|.{{GPUBindGroupDescriptor/layout}}.
+    This requires that its |bindingDescriptor|.{{GPUBindGroupLayoutBinding/type}} should be {{GPUBindingType/sampler}}.
+    And its |bindingDescriptor|.{{GPUBindGroupLayoutBinding/binding}} should equal to |bindingDescriptor|.{{GPUBindGroupBinding/binding}}
+
+<dfn>texture view binding validation</dfn>: A corresponding texture binding should exist in {{GPUBindGroupLayoutDescriptor/bindings}} of |descriptor|.{{GPUBindGroupDescriptor/layout}}.
+    This requires that its |bindingDescriptor|.{{GPUBindGroupLayoutBinding/type}} should be either {{GPUBindingType/sampled-texture}} or {{GPUBindingType/storage-texture}}.
+    And its |bindingDescriptor|.{{GPUBindGroupLayoutBinding/binding}} should equal to |bindingDescriptor|.{{GPUBindGroupBinding/binding}}
+    Its |bindingDescriptor|.{{GPUBindGroupLayoutBinding/textureDimension}} should be the same with |bindingDescriptor|.{{GPUTextureViewDescriptor/dimension}}.
+    Its |bindingDescriptor|.{{GPUBindGroupLayoutBinding/textureComponentType}} should be compatible with |bindingDescriptor|.{{GPUTextureViewDescriptor/format}}.
+    If |bindingDescriptor|.{{GPUBindGroupLayoutBinding/multisampled}} is `true`, the corresponding texture's |bindingDescriptor|.{{GPUTextureDescriptor/sampleCount}} should be greater than 1.
+    Otherwise, if |bindingDescriptor|.{{GPUBindGroupLayoutBinding/multisampled}} is `false`, the corresponding texture's |bindingDescriptor|.{{GPUTextureDescriptor/sampleCount}} should be 1.
+    If |bindingDescriptor|.{{GPUBindGroupLayoutBinding/type}} is {{GPUBindingType/sampled-texture}}, the corresponding texture's {{GPUTextureDescriptor/usage}} should include {{GPUTextureUsage/SAMPLED}}.
```

Comment:
nit:
```suggestion
    If |bindingDescriptor|.{{GPUBindGroupLayoutBinding/type}} is {{GPUBindingType/"sampled-texture"}}, the corresponding texture's {{GPUTextureDescriptor/usage}} should include {{GPUTextureUsage/SAMPLED}}.
```
so the quotes appear in the render output.

Reply:
Hmm. I revised a few more places where they are constants of strings.